### PR TITLE
style(docs): side bar style update

### PR DESCRIPTION
**Pull request by simeng-li**
## Update side bar styling according to the latest design

[figma](https://www.figma.com/file/u5rDl2y4PKPpiXqwYVWKn6/Documentation-website?node-id=0%3A1)

New Update:

image

image

test locally
@logto-io/eng

### CODE REVIEW

File: src/scss/custom.scss [hunk 1]
```diff
@@ -59,10 +62,123 @@
...
+  font-weight: 600;
+}
+
+.sidebar-section:not(:first-child) {
+  margin-top: calc(var(--ifm-menu-link-padding-vertical) * 2);
+}
+
+.menu__list {
+  .menu__list {
+    padding-left: 0;
```
Comment (gao-sun) [inline src/scss/custom.scss:80]: or use negative margin? don't know if the hover state looks good

File: src/scss/custom.scss [hunk 2]
```diff
@@ -59,10 +62,123 @@
...
+  }
+}
+
 .menu__link {
-  border-radius: 8px;
+  word-break: break-all;
```
Comment (gao-sun) [inline src/scss/custom.scss:85]: don't know if we should add this. i'm afraid of breaking something not properly

File: src/scss/custom.scss [hunk 3]
```diff
@@ -19,9 +19,10 @@
   --ifm-hover-overlay: rgba(93, 52, 242, 0.12);
   --ifm-code-font-size: 95%;
   --ifm-navbar-height: 64px;
-  --doc-sidebar-width: 400px !important;
+  --doc-sidebar-width:340px !important;
   --doc-sidebar-section-title: #c9c5d0;
   --logto-border: #c4c7c7;
+  --ifm-menu-link-sublist-icon: url('/static/img/side-bar-error.svg');
```
Comment (gao-sun) [inline src/scss/custom.scss:25]: file name looks weird. maybe `sidebar-collapse.svg`?

Comment (simeng-li) [reply src/scss/custom.scss:25]: good point no idea where does this name come from....

File: src/scss/custom.scss (hunk 2 above)
Comment (simeng-li) [reply src/scss/custom.scss:85]: Per design, sidebar width is fixed and should break to the next line if exceeds the length.

Design:
image

If not set to break all:
image

File: src/scss/custom.scss (hunk 1 above)
Comment (simeng-li) [reply src/scss/custom.scss]: Check its structure, feels the same but updated.

File: src/scss/custom.scss (hunk 2 above)
Comment (gao-sun) [reply src/scss/custom.scss:85]: i'm ok with both. let's confirm with design

File: src/scss/custom.scss (hunk 3 above)
Comment (gao-sun) [reply src/scss/custom.scss]: lul

**Review by gao-sun — APPROVED** [review]

**Review by wangsijie — APPROVED** [review]

File: src/scss/custom.scss (hunk 2 above)
Comment (simeng-li) [reply src/scss/custom.scss:91]: checked

**Pull request merged**